app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
+  def recent_posts
+    posts.count(3)
+  end
```

Comment:
- [ ] We want to return the most recent posts so that means we would have to implement some of **order** for fetching based on that. Kindly implement that. See suggestion below:

  ```
  def recent_post
    posts.order('created_at Desc').limit(3)
  end
  ```

Reply:
well resolved! ✔️